vhdbuilder/packer/cleanup.sh

```diff
@@ -59,6 +60,45 @@ if [[ -n "${IMPORTED_IMAGE_NAME}" ]]; then
   fi
 fi
 
+#cleanup managed image sig image version
+if [[ -n "${SIG_IMAGE_NAME}" ]]; then
+   echo "SIG_IMAGE_NAME is ${SIG_IMAGE_NAME}"
+   MANAGED_IMAGE_SIG_NAME=${SIG_IMAGE_NAME}
```

Comment:
In sigMode, SIG_IMAGE_NAME is the final created sig image name for test and should not be deleted.

Reply:
What about the gen2Mode? (the one that matches with Linux) GEN2_SIG_IMAGE_NAME has already been converted to VHD. The steps: `run-test` => `convert` => `cleanup` in the current pipeline. Should we delete it?

For gen2SigMode, I think the final created sig should be kept there?

Reply:
for gen2Mode, the built sig image should be deleted since we only need to upload the vhd to the storage account after converting the sig to vhd.
for gen2SigMode, we need to keep it. But we need to use the provide gallery name, image name and image version.